src/assets/scss/_variables.scss

```
@@ -0,0 +1,14 @@
+$enable-gradients:            true;
+// $body-bg:                   $white;
+$blue:    #c8d8f0 !default;
```

```suggestion
$blue:    #c8d8f0;
```
Remove all default flags, you are overriding these defaults.

---

I will remove these, but are still a bit uncertain which I'm going to need.